app/fighters/knight.py

```diff
@@ -0,0 +1,59 @@
+from app.fighters.tools.poiton import Potion
+from app.fighters.tools.weapon import Weapon
+from app.fighters.tools.armour import Armour
+
+
+class Knight:
+    def __init__(self, name: str, power: int, hp: int,
+                 armour: list[Armour], weapon: Weapon,
+                 potion: Potion | None, protection: int = 0) -> None:
```

Comment:
Read this message please and fix everywhere
https://mate.academy/chats/244801?selectedMessageId=3129519